qa/rpc-tests/wallet_shieldcoinbase_ua_nu5.py

```diff
@@ -15,17 +15,15 @@ def test_init_zaddr(self, node):
         # this function may be called no more than once
         assert(self.account is None)
         self.account = node.z_getnewaccount()['account']
-        self.addr = node.z_getaddressforaccount(self.account)['unifiedaddress']
+        self.addr = node.z_getaddressforaccount(self.account)['address']
         return self.addr
 
     def test_check_balance_zaddr(self, node, expected):
         balances = node.z_getbalanceforaccount(self.account)
         assert('transparent' not in balances['pools'])
         assert('sprout' not in balances['pools'])
-        # assert('sapling' not in balances['pools'])
         sapling_balance = balances['pools']['sapling']['valueZat']
         assert_equal(sapling_balance, expected * COIN)
-        # assert_equal(balances['pools']['orchard']['valueZat'], expected * COIN)
```

Comment:
Hmm, these commented-out lines were supposed to be uncommented once we got Orchard support working, and we were going to remember to do this because the test was _supposed_ to start failing once Orchard was integrated and the test started selecting Orchard as the recipient. We need to investigate why it didn't start failing, and fix the test to actually send to Orchard (which it is supposed to do as this is the NU5 variant of the UA test).

Reply:
asyncrpcoperation_shieldcoinbase currently just always selects the Sapling receiver for shielding. So these lines should be restored, but they can't be uncommented yet. https://github.com/zcash/zcash/issues/5511 is the issue tracking this.